File: 8-semester-pamo/exercise-2/Tipper/app/src/main/java/com/example/tipper/Menu.java
// Sandro Sobczyński (s20600)

package com.example.tipper;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class Menu extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.menu);

        Button bmiButton = findViewById(R.id.bmiButton);
        bmiButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Menu.this, BMI.class);
                startActivity(intent);
            }
        });

        Button caloriesButton = findViewById(R.id.caloriesButton);
        caloriesButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Menu.this, Calories.class);
                startActivity(intent);
            }
        });

        Button eatButton = findViewById(R.id.eatButton);
        eatButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Menu.this, Eat.class);
                startActivity(intent);
            }
        });

        Button returnButton = findViewById(R.id.returnButton);
        returnButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Menu.this, Start.class);
                startActivity(intent);
            }
        });
    }

}

File: 8-semester-pamo/exercise-4/Tipper/app/src/main/java/com/example/tipper/Menu.kt
// Sandro Sobczyński (s20600)
package com.example.tipper

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class Menu : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.menu)
        val bmiButton = findViewById<Button>(R.id.bmiButton)
        bmiButton.setOnClickListener {
            val intent = Intent(this@Menu, BMI::class.java)
            startActivity(intent)
        }
        val caloriesButton = findViewById<Button>(R.id.caloriesButton)
        caloriesButton.setOnClickListener {
            val intent = Intent(this@Menu, Calories::class.java)
            startActivity(intent)
        }
        val eatButton = findViewById<Button>(R.id.eatButton)
        eatButton.setOnClickListener {
            val intent = Intent(this@Menu, Eat::class.java)
            startActivity(intent)
        }
        val quizButton = findViewById<Button>(R.id.quizButton)
        quizButton.setOnClickListener {
            val intent = Intent(this@Menu, HealthyDietQuiz::class.java)
            startActivity(intent)
        }
        val chartButton = findViewById<Button>(R.id.chartButton)
        chartButton.setOnClickListener {
            val intent = Intent(this@Menu, BMIChart::class.java)
            startActivity(intent)
        }
        val returnButton = findViewById<Button>(R.id.returnButton)
        returnButton.setOnClickListener {
            val intent = Intent(this@Menu, Start::class.java)
            startActivity(intent)
        }
    }
}